src/components/Guilds/common/Modal/transaction.tsx

```diff
@@ -0,0 +1,128 @@
+import PendingCircle from 'components/common/PendingCircle';
+import { Modal, ModalProps } from 'components/Modal';
+
+import styled from 'styled-components';
+
+type TransasctionModalProps = Pick<ModalProps, 'isOpen' | 'onDismiss' | 'onCancel'>;
+
+export const Container = styled.div`
+  display: flex;
+
+  flex-direction: column;
+
+  align-items: center;
+
+  padding: 24px;
```

Comment:
spaces between lines? . Colors variables should be theme.

Reply:
im not sure why yarn formatted it like that. I'll fix it manually at the end.